src/test/java/uk/gov/pay/directdebit/payments/fixtures/GoCardlessEventFixture.java

```diff
@@ -109,6 +125,141 @@ public GoCardlessEventFixture withOrganisationIdentifier(GoCardlessOrganisationI
         return this;
     }
 
+    public String getDetailsCause() {
+        return detailsCause;
+    }
+
+    public GoCardlessEventFixture setDetailsCause(String detailsCause) {
```

Comment:
For the setting methods on fixtures and builders we usually use the prefix `with` rather than `set` so it reads nicer:
```
aGoCardlessEventFixture
.withDetailsCause("somecause")
.withDetailsDescription("some description);
```